src/SFA.DAS.Campaign.Web/ViewComponents/TrainingProvider-Heading/TrainingProviderHeadingViewComponent.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Sfa.Das.Sas.ApplicationServices.Models;
+using Sfa.Das.Sas.Shared.Components.Orchestrators;
+using Sfa.Das.Sas.Shared.Components.ViewComponents.TrainingProvider.Search;
+using Sfa.Das.Sas.Shared.Components.ViewModels;
+
+namespace SFA.DAS.Campaign.Web.ViewComponents
+{
+    public class TrainingProviderHeadingViewComponent : ViewComponent
+    {
+        private readonly IApprenticeshipOrchestrator _apprenticeshipOrchestrator;
+        private readonly ITrainingProviderOrchestrator _trainingProviderOrchestrator;
+
+        public TrainingProviderHeadingViewComponent(IApprenticeshipOrchestrator apprenticeshipOrchestrator, ITrainingProviderOrchestrator trainingProviderOrchestrator)
+        {
+            _apprenticeshipOrchestrator = apprenticeshipOrchestrator;
+            _trainingProviderOrchestrator = trainingProviderOrchestrator;
+        }
+
+        public async Task<IViewComponentResult> InvokeAsync(TrainingProviderSearchViewModel queryModel)
+        {
+
+            var apprenticeshipType = _apprenticeshipOrchestrator.GetApprenticeshipType(queryModel.ApprenticeshipId);
+
+            var model = new TrainingProviderHeadingViewModel();
+
+            model.SearchResults = await _trainingProviderOrchestrator.GetSearchResults(queryModel);
```

Comment:
The use of the Orchestrators from the Shared package feel a little odd being used here. It feels like we're directly referencing the internals of the view components.

We're also making 2 calls to the provider search aren't we? One from here and one from the ProviderSearchResults View Component in the shared package.

Is this not a candidate to be view component in the shared package?